app/facades/foods_facade.rb

```diff
@@ -0,0 +1,15 @@
+class FoodsFacade 
+    def get_foods_search(search)  
+        get_foods_data(search)[:foods].map do |data|
+            Food.new(data)
+        end
+    end 
```

Comment:
The facade would be a great place to handle the responsibility of only returning 10 Food results. We can also get both the `total_hits` with this call, something like: 
```
def get_foods_search(search)
  results = get_foods_data(search)
  total_hits = results[:totalHits]
  foods = results[:foods].first(10).map do |data|
    Food.new(data)
  end
  [total_hits, foods]
end
```
This way, you wouldn't be making extraneous calls to the API, and you wouldn't create unused instances of Food.